packages/bundler-webpack/src/build/ssr/createClientPlugin.ts

```diff
@@ -51,22 +52,25 @@ export const createClientPlugin = (
 
           // get asset modules
           const assetModules = modules.filter(
-            (m): m is FnModules & Required<Pick<FnModules, 'assets'>> =>
-              !!(m.assets && m.assets.length),
+            (
+              module,
```

Comment:
We'd better avoid using`module` keyword as var name. That's why we use `m` previously